azure/notebooks/Azure-MNMG-RF.ipynb

```diff
@@ -8,7 +8,7 @@
     "\n",
```

Comment:
<p>This will require a step to accept the EULA for using this image, I used the Azure CLI:&nbsp;<a href="https://docs.microsoft.com/en-us/cli/azure/vm/image/terms?view=azure-cli-latest" target="_blank">https://docs.microsoft.com/en-us/cli/azure/vm/image/terms?view=azure-cli-latest</a>, can you add the instructions?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>